docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
     image: node:14-alpine
     environment:
       APPEALS_SERVICE_API_URL: http://appeals-service-api:3000
+      SESSION_KEY: some_secure_key_goes_here
```

Comment:
not sure if this is the best place for this.

I'm using the session to keep track of the current `uuid` for the "in progress" form submission

Reply:
I like that. Makes it nice and easy for me to generate a random secret in the deployment